Makefile

```diff
@@ -175,13 +175,16 @@ prow-release-controller-definitions:
 	oc annotate -n openshift is/origin-v4.0 "release.openshift.io/config=$$(cat ci-operator/infra/openshift/release-controller/origin-4.0.json)" --overwrite
 	oc annotate -n ocp is/4.0-art-latest "release.openshift.io/config=$$(cat ci-operator/infra/openshift/release-controller/ocp-4.0.json)" --overwrite
 	oc annotate -n ocp is/4.0 "release.openshift.io/config=$$(cat ci-operator/infra/openshift/release-controller/ocp-4.0-ci.json)" --overwrite
+	oc annotate -n ocp is/4.1-art-latest "release.openshift.io/config=$$(cat ci-operator/infra/openshift/release-controller/ocp-4.1.json)" --overwrite
```

Comment:
Both image streams need to be created as well.

Reply:
Not sure what you mean.

Reply:
`is/4.1-art-latest` already exists. We pushed to it recently :-) As does `is/4.1` there's just nothing in it yet.

```$ oc get is -n ocp 4.1-art-latest | head
NAME             DOCKER REPO                                        TAGS                                                            UPDATED
4.1-art-latest   registry.svc.ci.openshift.org/ocp/4.1-art-latest   console,logging-elasticsearch5,logging-curator5 + 106 more...   4 hours ago
[~] 14:57:07 
$ oc get is -n ocp 4.1 | head           
NAME   DOCKER REPO                             TAGS   UPDATED
4.1    registry.svc.ci.openshift.org/ocp/4.1          
```

Reply:
I'll do it in a follow up.